### PR TITLE
Ver 1x/m235 p1

### PR DESCRIPTION
**Pull request by kamil-jagoda**

### CODE REVIEW

File: Magento_Catalog/templates/product/view/options/type/text.phtml [hunk 1]
```diff
@@ -45,25 +45,38 @@ if ($_option->getMaxCharacters()) {
...
                 <?= $_option->getMaxCharacters() ?>
             </strong>
         </p>
     <?php endif; ?>
+    <?php if ($_option->getMaxCharacters()) :?>
+        <script type="text/x-magento-init">
+        {
+            "[data-selector='options[<?= $block->escapeJs($_option->getId()) ?>]']": {
+                "Magento_Catalog/js/product/remaining-characters": {
+                    "maxLength":  "<?= (int)$_option->getMaxCharacters() ?>",
```
[the remark this answers is not in the document]
Comment (kamil-jagoda) [reply Magento_Catalog/templates/product/view/options/type/text.phtml:74]: on gitlab it showed as a bug if it wasn't added

File: Magento_Checkout/web/template/billing-address/actions.html [hunk 2]
```diff
@@ -0,0 +1,21 @@
+<!--
```
[the remark this answers is not in the document]
Comment (kamil-jagoda) [reply Magento_Checkout/web/template/billing-address/actions.html]: probably you didn't see last commit with changes, for this file was 3 commits: init, formatting and last with changes

File: Magento_Checkout/web/template/billing-address.html [hunk 3]
```diff
@@ -1,55 +1,44 @@
...
             class="checkbox__field"
             data-bind="
-                checked: isAddressSameAsShipping,
-                click: useShippingAddress,
-                attr: {
-                    id: 'billing-address-same-as-shipping-' + getCode($parent)
-                }
+                checked: isAddressSameAsShipping, 
+                click: useShippingAddress, 
+                attr: {id: 'billing-address-same-as-shipping-' + getCode($parent)}
```
Comment (Januszpl) [inline Magento_Checkout/web/template/billing-address.html:18]: why changing this part?

File: Magento_Checkout/web/template/billing-address.html [hunk 4]
```diff
@@ -1,55 +1,44 @@
...
             <use data-bind="attr: { 'xlink:href': require.toUrl('images/icons-sprite.svg#checked') }" xlink:href=''></use>
         </svg>
+
         <label
             class="checkbox__label"
             data-bind="
-                attr: {
-                    for: 'billing-address-same-as-shipping-' + getCode($parent)
-                }
+                attr: {for: 'billing-address-same-as-shipping-' + getCode($parent)}
```
Comment (Januszpl) [inline Magento_Checkout/web/template/billing-address.html:28]: the same here (old formatting was ok)

File: Magento_Checkout/web/template/billing-address.html [hunk 5]
```diff
@@ -1,55 +1,44 @@
...
             "
         >
-            <span
-                data-bind="i18n: 'My billing and shipping address are the same'"
-            ></span>
+            <span data-bind="i18n: 'My billing and shipping address are the same'"></span>
```
Comment (Januszpl) [inline Magento_Checkout/web/template/billing-address.html:31]: here as well old formatting was ok

**Pull request merged**